bundles/redhat8.10/packages.txt.gotmpl

```diff
@@ -21,21 +21,17 @@ kubeadm-{{ .KubernetesVersion }}-0
 cri-tools
 {{- end }}
 conntrack
-ebtables
+iptables-ebtables
 ethtool
 iproute
 iptables
 iscsi-initiator-utils
-socat
 gcc
 glibc-devel
 glibc-gconv-extra
 make
 libseccomp
 iproute-tc
-gssproxy
```

Comment:
I think these were added for Nutanix CSI. I didnt wanted to two versions of the package list so I added it here too.

Reply:
let me double check the requirements

Reply:
these are required 

```   - iscsi-initiator-utils
      - nfs-utils
      - lvm2
      - xfsprogs
      - ipvsadm
      - lsscsi
      - sysstat
```
so we should be OK